TwoHoSun/Screens/Home/MainView/NotificationView.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  NotificationView.swift
+//  TwoHoSun
+//
+//  Created by 김민 on 10/20/23.
+//
+
+import SwiftUI
+
+struct NotificationView: View {
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+
+    var body: some View {
+        ZStack {
+            Color.white
+            notificationList
+        }
+        .navigationBarBackButtonHidden()
+        .navigationTitle("알림")
+        .navigationBarTitleDisplayMode(.inline)
+        .toolbar {
+            ToolbarItem(placement: .topBarLeading) {
+                backButton
+            }
+        }
+        .toolbarBackground(.white, for: .navigationBar)
```

Comment:
```suggestion
        .toolbarBackground(.white, for: .navigationBar)
       .toolbar(.hidden, for: .tabBar)
```
어떤 고민이 있는지 잘 살펴보았습니다. 탭바를 제거하기 위해서라면 앞에 있는 방법보다는 tabbar가 필요없는 뷰에서 hidden으로 해주는 좋은 방법을 알고 있어 공유드립니다.

Reply:
개쩐다!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Reply:
근데 이렇게 했을 때 pop 하면 tab view가 사라졌다 나오는 게 원래 맞나여?

Reply:
그러면 navigationStack을 메인탭뷰에서 관리를 해버리면 해결될거같습니다.

Reply:
그게 제가 제시한 방법이잔아요

Reply:
싸우지마요

Reply:
말투 예쁘게 바꿈.

Reply:
참고사항에서 언급한 navigation bar 오류가 어떤 오류에여?